web/src/features/auth/roles.ts

```diff
@@ -0,0 +1,5 @@
+export const ROLES = {
+  HFI: {
+    SET_FIRE_STARTS: 'hfi_set_fire_starts'
```

Comment:
I guess we could load this from the environment too - but that seems excessive.

Reply:
Feels excessive to me -- I also feel our set of environment variables is already excessive and unwieldy.

Reply:
Agreed